dts/bindings/net/wireless/nordic,nrf-radio.yaml

```diff
@@ -38,6 +38,17 @@ properties:
         Value zero means no antenna available. If provided value different than
         zero, it must be greater or equal to two.
 
+    dfe-pdu-ant:
```

Comment:
I don't know what an antenna switch pattern is, but I wonder if the name of the property should be something like `dfe-ant-switch-pattern` or something like that?

Reply:
knowing that there also exists an `ANT+` wireless protocol, then I think using `ant` for antenna in a `net/wireless/nordic` file is bad naming.

Why is `antenna` so long that is has to be abbreviated ?

Reply:
@tejlmand I'll change the name to `dfe-pdu-antenna`.

@mbolivar-nordic 
The `dfe-ant-switch-pattern` will not inform about actual purpose of the property. Direction Finding extension in Nordic Radio uses `switch-patterns` to drive hardware switches through GPIOs. That functionality is required to use multiple antennas when TX/RX Constant Tone Extension. 
There is max 40 `switch-patterns` stored in DFE. First of them is used by Radio peripheral (through DFE) to select antenna to TX/RX Bluetooth PDU (PDU that has attached CTE). BT spec. does not say anything about PDU antenna, so it is Nordic proprietary thing. Due to that I decided provide end user possibility to set PDU `switch-pattern` in DTS. It seems to me the best place to put it.

Reply:
BTW I'll change `dfe-ant-num` to `dfe-antenna-num` to keep the same naming style.

Reply:
> The `dfe-ant-switch-pattern` will not inform about actual purpose of the property.

Thanks for confirming.